packages/inter-protocol/src/clientSupport.js

```diff
@@ -239,17 +236,20 @@ const makeBidOffer = (brands, opts) => {
   const offerArgs =
     'price' in opts
       ? {
-          want,
+          // XXX "targetPurchase" or "max" might be better names
+          // for what this does, but for now, here we are.
```

Comment:
this is conflating the arg and offer safety (Zoe) wants. I thought we landed on `desiredBuy` as the argument and being able to define it separately from the Zoe want.

I'll defer to @Chris-Hibbert on whether this should be addressed before merge.

Reply:
I'd like the CLI commands to reflect the distinction even if the names haven't been updated in the contract yet.  `desiredBuy` is what the user should see.  

The simplest place to describe what I'm hoping for was in the .md file on command usage.

Reply:
ok. done.